### DISCUSSION TITLE
Unable to find the system's preset individuals template

### DISCUSSION THREAD

**Discussion by AD-Vina** [Open-Systems-Pharmacology/Forum#1094] [Questions & Problems] [answered]
I am learning how to use PK-sim software, when I download and install PK-sim and open it, right click on Idividuals and select "load from templates", the system says "No template 'individual' available in the template database". But I found out that normally there should be 'Standard European Male for DDI', so I am very confused and hope to get help!

**Comment by AD-Vina** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/82560295/151094997-c43d2c8a-1138-4de4-9797-32bdda1f8b8c.png)

**Comment by msevestre** [comment]
Hi @AD-Vina 
What do you mean with 
>But I found out that normally there should be 'Standard European Male for DDI', so I am very confused and hope to get help!

At any rate, those predefined templates where removed in v10 from the database so your installation is working perfectly. You can create an individual in a few clicks as outlined in the documentation (see https://docs.open-systems-pharmacology.org).

You can also load some predefined compound templates. This should work with your installation out of the box.

Hope that helps

**Reply by AD-Vina** [reply]
Thank you!
